### PR TITLE
Form refactoring

### PR DESCRIPTION
**Pull request by evgeniush**

### CODE REVIEW

**Comment by 1982irushka** [comment]
when all form fields are filled and button is clicked, the form is not submitted.

**Comment by evgeniush** [comment]
> when all form fields are filled and button is clicked, the form is not submitted.

Fixed

File: .eslintrc.json [hunk 1]
```diff
@@ -22,6 +22,12 @@
       {
         "js": "always"
       }
+    ],
```
Comment (evgeniush) [inline .eslintrc.json:25]: Test comment

Comment (evgeniush) [reply .eslintrc.json:25]: Answer test

**Review by 1982irushka — APPROVED** [review]

**Pull request merged**